vctr90/expenses-manager-api
# Chrore/34 refactor code for testing purposes

### Description

This PR seeks to change server modules so they all use dependency injection for better testing proposes.

### Isues/tasks it solves

Issue #36 

### Impacted areas

* All modules of the application

### How to test

For now just run `npm test` and make sure all automated unit tests are passing (integration tests are on the way)

### Migrations

NO
